config/metadata/rulesMeta.yaml

```
@@ -615,14 +615,18 @@ groups:
           
           The field can also include a prefix that changes the span used for evaluation of the field.
           The only prefix currently supported is `root`, as in `root.http.status`. Specifying `root.`
-          causes the condition to be evaluated against the root span. 
+          causes the condition to be evaluated against the root span.
           
           For example, if the `Field` is `root.url`, then the condition will be processed using the url
-          field from the root span. 
+          field from the root span.
           
           The setting `Scope: span` for a rule does not change the meaning of this prefix --
           the condition is still evaluated on the root span and is treated as if it were part of the 
-          span being processed. 
+          span being processed.
+          
+          When using the `not-exists` operator to check a condition with a `root.` prefix, if the root
+          span does not exist on the trace, then the condition will evaluate to false.
```

What do you think of including a common reason why a root span would not exist on the trace in Refinery?

```suggestion
          span does not exist on the trace, as in the case when a root span is taking a long time to complete and does not arrive before a sampling decision is being made, then the condition will evaluate to false.
```